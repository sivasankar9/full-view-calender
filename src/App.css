@import '~@fullcalendar/core/main.css';
@import '~@fullcalendar/daygrid/main.css';

.button{
  font-style: italic;
}
body{
  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
}