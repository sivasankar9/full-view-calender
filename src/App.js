import React,{Component} from 'react';
import PropTypes from 'prop-types';

const upper = (a)=>{a.toUpperCase()};
class App extends Component{

    render(){
        return (<div>namez:::{upper('hi all')} Welcome siva sankar, manasa & Roith.
            </div>);
    }
}

App.propTypes = {
  namez: PropTypes.string.isRequired
}

export default App;
