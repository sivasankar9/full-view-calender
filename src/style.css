.button{
  font-style: italic;
}
body{
  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
}