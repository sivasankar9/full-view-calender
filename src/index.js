import './style.css';
import {applyMiddleware, combineReducers, compose, createStore} from 'redux';
import App from './App';
import {Provider} from 'react-redux';
import React from 'react';
import fullCalender from './components/full-calender/reducer';
import loginCalender from './components/full-calender/components/shared/login/reducer';
import registerFormCalender from './components/full-calender/components/shared/register-form/reducer';
import { render } from 'react-dom';
import thunkMiddleware from 'redux-thunk';
 

let store = createStore(combineReducers({
    fullCalender,
    loginCalender,
    registerFormCalender
}), compose(
      applyMiddleware(thunkMiddleware),
      window.devToolsExtension ? window.devToolsExtension() : f => f
    ));


render(<Provider store = {store}>
    <App />
    </Provider>, document.getElementById('root'));
