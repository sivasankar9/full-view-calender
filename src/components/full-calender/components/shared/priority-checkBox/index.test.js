import Chance from 'chance';
import PriorityCheckBox from './index';
import React from 'react';
import {shallow } from '../../../../../../src/enzyme';


describe('Given the <priorityCheckBox/> component', () => {
    
    const chance = new Chance();

    let priorityCheckBox;

    const testProps = Object.freeze({
        label: chance.word(),
         handlerPriority: f=>f, 
         value: chance.word(), 
         isSelected: chance.bool()
        });

    beforeEach(() => {

        priorityCheckBox = shallow(<PriorityCheckBox {...testProps} />);

    });

    it('should match the snapshot', () => {

        expect(priorityCheckBox).toMatchSnapshot();

    });

});
