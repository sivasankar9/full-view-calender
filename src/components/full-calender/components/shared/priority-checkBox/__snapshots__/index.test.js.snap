// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the <priorityCheckBox/> component should match the snapshot 1`] = `ShallowWrapper {}`;
